### PR TITLE
Added the toolbar feature to the project as well as the headline page

### PR DESCRIPTION
**Pull request by Tchilo**
Added the toolbar feature to the project 
Added the headline page to the project 
Added a screenshot of the project

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    background-color: #1c1a19;
+    color: white;
+    background-image: url(/images/Group.png);
+    background-repeat: no-repeat;
+    background-position: right bottom;
+    padding-top: 120px;
+    height: 85vh;
+  }
+  
+  .headline-page h1 {
```
Comment (jubaan) [inline style.css]: - [ ] BAD `.headline-page h1` Good `.main-title`

File: style.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    font-family: Crete Round, sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 56px;
+    line-height: 64px;
+    letter-spacing: 0.37px;
+    margin: 12px;
+  }
+  
+  .headline-page h2 {
```
Comment (jubaan) [inline style.css]: - [ ] BAD `.headline-page h2` Good `.subtitle`

File: style.css [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    font-family: roboto, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 24px;
+    line-height: 32px;
+    color: #fff5e1;
+    margin: 12px;
+  }
+  
+  .headline-page p {
```
Comment (jubaan) [inline style.css]: - [ ] BAD `.headline-page p` Good `.headline-content`

**Review by jubaan — CHANGES_REQUESTED** [review]
Hi @Tchilo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+          <a href="#"><i class="fa fa-angellist clr-secondary"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa fa-medium"></i></a>
+        </li>
+      </ul>
+    </section>
```
Comment (Temmarie) [inline index.html:47]: - [ ] You must stick to the design as much as possible (e.g., font, colours, images, tests, margins) using the templates in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Look out for the following :
- The fonts,
- The toolbar/navigation bar 
- The Image sizes and placement.
_See the images below for comparisons_

**Original** | **Your clone**
  ----------- | -----------
 image | image

Comment (DeliceLydia) [reply index.html]: - [ ] According to the requirements you need to stick to the design as much as possible(fonts, colors, etc)
tmp

If you can compare your portfolio and the design it's really different, that is why I encourage you to work on it again and don't give up take this as learning!

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Tchilo , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._


### Keep Your Head Up High!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

N.B: Because this review was the third as described in the requirements you are required to start the project again! and please don't give up!
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi @Tchilo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted


## Optional suggestions to improve your project 🥇 

Please, consider:
- [OPTIONAL] Use [Color Cop](http://colorcop.net/download/) to take the exactly hexadecimal code to each color, in that way your CSS style could improve this.
- [OPTIONAL] [Google fonts](https://fonts.google.com/) is another tool could help you to get a better typography

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**